ydb/docs/en/core/integrations/orm/linq2db.md

```diff
@@ -0,0 +1,501 @@
+# {{ ydb-short-name }} Provider for LinqToDB
+
+## Overview {#overview}
+
+This guide explains how to use [LinqToDB](https://linq2db.github.io/) with {{ ydb-short-name }}.
+
+LinqToDB is a lightweight and fast ORM/µ-ORM for .NET that provides type-safe LINQ queries and precise SQL control. The {{ ydb-short-name }} provider generates correct YQL, supports YDB types, schema generation, and bulk operations (Bulk Copy).
```

Comment:
Let's make LINQ a link to https://en.wikipedia.org/wiki/Language_Integrated_Query or https://learn.microsoft.com/en-us/dotnet/csharp/linq/ or